tlf/id.go

```diff
@@ -22,8 +23,38 @@ const (
 	idSuffix = 0x16
 	// pubIDSuffix is the last byte of a public top-level folder ID
 	pubIDSuffix = 0x17
+	// singleTeamIDSuffix is the last byte of a single-team top-level
+	// folder ID
+	singleTeamIDSuffix = 0x26
 )
 
+// Type is the type of TLF represented by a particular ID (e.g.,
+// public, private, etc.)
+type Type int
+
+const (
+	_ Type = iota
+	// Private represents a private TLF between one or more individual users.
+	Private
+	// Public represents a public TLF for one or more individual users.
+	Public
+	// SingleTeam represents a private TLF for a single Keybase team.
+	SingleTeam
+)
+
+func (t Type) String() string {
+	switch t {
+	case Private:
+		return "private"
+	case Public:
+		return "public"
+	case SingleTeam:
+		return "singleTeam"
```

Comment:
I'd rather we use snake case for these strings: `single_team`, but I won't argue past just making that assertion :)